test/testData/mockReportData.json

```diff
@@ -251,7 +260,119 @@
         "kbCategory": "test kb category",
         "msiScore": 0.0
     },
+    "kbMatchedStatements": [
+        {
+            "approvedTherapy": false,
+            "category": "therapeutic",
+            "context": "test multivariant statement",
+            "disease": "colorectal cancer [DOID:9256]",
+            "kbStatementId": "#999:999",
+            "matchedCancer": true,
+            "reference": "pmid:TEST1",
+            "relevance": "resistance",
+            "iprEvidenceLevel": "IPR-D",
+            "externalSource": "IPRKB",
+            "reviewStatus": "pending",
+            "kbData": {},
+            "kbMatches": [
+                {
+                    "kbVariant": "APC:p.T1556fs",
+                    "variant": "a12458ff7c8eb865656fd61f670cd0d5",
+                    "variantType": "mut",
+                    "kbVariantId": "#158:1343"
+                },
+                {
+                    "kbVariant": "ZFP36L2:p.Q401del",
+                    "variant": "TEST1",
+                    "variantType": "mut",
+                    "kbVariantId": "#999:9999"
+                }
+            ]
```

Comment:
Is kbMatchedStatements only for fully-matched statements? And kbMatchedStatements.kbMatches an array where each element correspond to an element in kbMatches?

Reply:
Each element in the kbmatch array will be created as a new kbmatch record in the db. It doesn't have to be defined anywhere else in the json (except the observed variant of course).